### DISCUSSION TITLE
Flux

### DISCUSSION THREAD

**Discussion by SweatyPug** [lllyasviel/Fooocus#3538] [Q&A] [answered]
Morning all.

Would it be possible to run Flux on fooocus? I'm definitely no pro, so I'm unsure if its possible yet.

Thanks in advance, Adam

**Comment by IPv6** [comment, marked as answer]
No, it is not possible.

**Reply by wisdombox44** [reply]
Try Forgewebui or SwarmUI

**Reply by SweatyPug** [reply]
Ok thank you.

**Reply by coreshock** [reply]
> No, it is not possible.

Or better try https://github.com/lllyasviel/stable-diffusion-webui-forge/ from the the same author. You can install Forge easily using Stability Matrix.

**Comment by DavidDragonsage** [comment]
SimpleSDXL is the only branch of Fooocus that has been updated to version 2.5.5 and is still in active development. It support Flux and SD3.

See this article:
https://github.com/lllyasviel/Fooocus/discussions/3548

**Reply by Aerthlyomi** [reply]
Can't make it work at the moment, but I am working on that. This could be exactly what I was looking for!
Thanks a lot!

**Reply by DavidDragonsage** [reply]
Yes I found installation tricky also. I have an installation guide at Pure Fooocus, if that will help:
https://www.facebook.com/groups/fooocus
